### PR TITLE
Fix top nav dark mode color 

### LINKED ISSUE #1043: Balance and wallet name difficult to read in dark mode
The wallet balance and wallet name are difficult to read in dark mode.
See the image below:

image

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #1043 

This pr fix text color on the top nav when in dark mode
replace hidden icon button with image icon
clean up hidden icon lines of code 

image

### CODE REVIEW

File: ui/page/main_page.go [hunk 1]
```diff
@@ -89,6 +82,10 @@ type MainPage struct {
 	dcrUsdtBittrex load.DCRUSDTBittrex
 	totalBalance   dcrutil.Amount
 
+	// hideBalance options
+	hideBalanceButton *decredmaterial.Clickable
+	tooltip           *decredmaterial.Tooltip
```
Comment (JustinBeBoy) [inline ui/page/main_page.go]: tooltip not use in this page so need to remove it

File: ui/page/main_page.go [hunk 2]
```diff
@@ -105,10 +102,9 @@ func NewMainPage(l *load.Load) *MainPage {
 		checkBox:   l.Theme.CheckBox(new(widget.Bool), "I am aware of the risk"),
 	}
 
-	mp.hideBalanceItem.hideBalanceButton = mp.Theme.IconButton(mp.Theme.Icons.ConcealIcon)
-	mp.hideBalanceItem.hideBalanceButton.Size = unit.Dp(19)
-	mp.hideBalanceItem.hideBalanceButton.Inset = layout.UniformInset(values.MarginPadding4)
-	mp.hideBalanceItem.tooltip = mp.Theme.Tooltip()
+	mp.hideBalanceButton = mp.Theme.NewClickable(true)
+	mp.hideBalanceButton.Radius = decredmaterial.Radius(7)
+	mp.tooltip = mp.Theme.Tooltip()
```
Comment (JustinBeBoy) [inline ui/page/main_page.go]: remove it

**Review by JustinBeBoy — CHANGES_REQUESTED** [review]

**Comment by JustinBeBoy** [comment]
please improve icon button show/hide balance when hover

image

**Review by dreacot — DISMISSED** [review]
As stated by @JustinBeBoy , the hover on the hide icon is flaky. I think we are good to merge once that is fixed

**Comment by Sirmorrison** [comment]
> please improve icon button show/hide balance when hover
> 
> image

The issue with the hover is the custom clickable... I will just have it removed. Another reason i am not using the icons, is that we are gradually migrating the icons used on the app to image icons and removing all material icons.

**Review by devchoplife — COMMENTED** [review]
Looks good
Please resolve conversations

**Review by JustinBeBoy — APPROVED** [review]

**Pull request merged**